### DISCUSSION TITLE
Can not acces power entity's

### DISCUSSION THREAD

**Discussion by MaxSchaaper** [hultenvp/solis-sensor#346] [Q&A]
Hello,
I have Homeassistant with a Solis integration working in docker containers. 
The HA energy dashboard is working fine. 
I also want to use the power entity's like the actual AC output power in Kw. 
The data is available as it can be selected in Influxdb.
Is there a way that i can get the actual power, as delivered every few seconds from soliscloud, to publish to my MQTT broker or any other solution.
I don't want to use the Riemann Sum option nor query's from a database.
Can anyone set me on the right track?
Thanks in advance.

**Comment by hultenvp** [comment]
Nope, not possible, Soliscloud is updated only every 2 mins. On top of that there might be 1 minute of jitter caused by poor timing of uploading from your inverter to the cloud (i.e. the inverter uploads exactly after you got the latest data in HA from the cloud).
